llvm/test/MC/RISCV/invalid-instruction-spellcheck.s

```diff
@@ -22,10 +22,10 @@ fl ft0, 0(sp)
 # CHECK-RV64IF: did you mean: flw, la, lb, ld, lh, li, lw
 # CHECK-NEXT: fl ft0, 0(sp)
 
-addd x1, x1, x1
```

Comment:
Why does this need to change in this patch? You didn't include `addd` in this patch.